### ISSUE TITLE
Launcher is not typing the email/password

### ISSUE THREAD

**Issue by blackr4bbit** [Healix/Gw2Launcher#232]
Since yesterday am having a problem when run the account in the launcher, theres execute the launcher but not type the email and the password, i try typing manually and press log in, it start normaly i close everything and start again but nothing happen same issue not typing the email and the password, any suggestion to fix this? i re-download the launcher but issue still happening.

**Comment by Healix** [comment]
I'm assuming you're not using the [beta version](https://github.com/Healix/Gw2Launcher/raw/master/Gw2Launcher/bin64/Beta/Gw2Launcher.exe)?

**Comment by blackr4bbit** [comment]
> I'm assuming you're not using the [beta version](https://github.com/Healix/Gw2Launcher/raw/master/Gw2Launcher/bin64/Beta/Gw2Launcher.exe)?

Thanks, i'll try the beta version and it work again, but, at executing the file, windows pop me up a advice that was unknow and possible dangeours file, i just ignore it, but its that normal?

**Comment by Healix** [comment]
Yes. All new versions will show that warning until enough people choose to allow it.

**Comment by Krasus1313** [comment]
I'm also having similar problem I write all information login but then I need to write it again. So I guess I also should use beta version?

And if I use it, do I need to set local.dats again or not since they're stored in somewhere.

**Comment by Healix** [comment]
GW2 is in read-only mode when you launch an account, so if you choose to remember your login, it's not going to save. You have to right click the account > Selected > Launch (normal).

The beta version is for when you choose to have Gw2Launcher remember your login and it fails to work. They upgraded to a new version of CoherentUI back in July that doesn't work well on some systems, which broke the release version. It also broke GW2's own remembered login for some, which causes the email and/or password to sometimes not appear.

**Comment by NamingIsHard4Me** [comment]
I got the same problem and using the beta version doesn't fix it. The problem appeared with the update today.

**Comment by Healix** [comment]
@NamingIsHard4Me 

That's because the patch switched from CoherentUI to CEF. Add -usecoherent to the arguments under Settings > Guild Wars 2 > Launch options.

**Comment by NamingIsHard4Me** [comment]
@Healix 

Works again. Tyvm <3
